labelme/app.py

```diff
@@ -41,7 +42,7 @@
 
 
 LABEL_COLORMAP = imgviz.label_colormap()
-
+GROUP_ID_COLORMAP = dict((key, tuple(np.random.randint(256, size=3))) for key in range(0, 100))
```

Comment:
I have a better function for this you can use that to create visually separable colors- 

```
def generateRandomColors(N, bright=False):
    """
    Generate random colors.
    To get visually distinct colors, generate them in HSV space then
    convert to RGB.
    """
    brightness = 1.0 if bright else 0.7
    hsv = [(i / N, 1, brightness) for i in range(N)]
    colors = list(map(lambda c: colorsys.hsv_to_rgb(*c), hsv))
    random.shuffle(colors)
    return colors
```